### DISCUSSION TITLE
I want to test the beta version of Kyuubi Web UI, what setup is required to test Kyuubi Web UI?

### DISCUSSION THREAD

**Discussion by phyyou** [apache/kyuubi#4760] [Q&A]
I'm interested in using out the beta version of Kyuubi Web UI and was wondering what setup is required to do so.
Specifically, I'd like to know what software and configurations are needed to get the Web UI up and running.

I've already installed Apache Kyuubi and have been able to run queries using the JDBC/ODBC interfaces. However, I'm not sure what additional steps are needed to set up the Web UI. Any guidance or pointers to relevant documentation would be greatly appreciated.

I'm currently using 1.7.0 now! Should I upgrade to the 1.8.0-master-snapshot?

@zwangsheng ? @zhaohehuhu ? @pan3793 ?

**Comment by zwangsheng** [comment]
Unfortunately, Kyuubi Web UI have not been perfected at present, which may be slightly crude and the corresponding deployment document is incomplete. If you want to follow the development progress of this piece and contributor in the development, I will create an Issue about the Web UI deployment document, you can follow that Issue and get the current status.

[[DOCS][UI] Deploy Kyuubi UI ](https://github.com/apache/kyuubi/issues/4761)

**Comment by pan3793** [comment]
Build binary tarball using `build/dist --web-ui`
```
Usage:
+----------------------------------------------------------------------------------------------+
| ./build/dist [--name <custom_name>] [--tgz] [--web-ui] [--flink-provided] [--hive-provided]  |
|              [--spark-provided] [--mvn <maven_executable>] <maven build options>             |
+----------------------------------------------------------------------------------------------+
name:           -  custom binary name, using project version if undefined
tgz:            -  whether to make a whole bundled package
web-ui:         -  whether to include web ui
flink-provided: -  whether to make a package without Flink binary
hive-provided:  -  whether to make a package without Hive binary
spark-provided: -  whether to make a package without Spark binary
mvn:            -  external maven executable location
```

And access `http://kyuubi:<rest-port>/ui`

**Reply by phyyou** [reply]
Thanks for the tip, I'll give it a try!
